setup.py

```diff
@@ -193,7 +193,7 @@ def run(self):
         sdist.run(self)
 
 
-extras_require = {"fits": ["astropy"], "gdal": ["gdal"], "simpleitk": ["SimpleITK"]}
+extras_require = {"fits": ["astropy"], "gdal": ["gdal"], "itk": ["itk"]}
```

Comment:
Out of curiousity, do you have some motivation for why itk is preferred over simpleitk here?

Reply:
`itk` is more standard and capable.